### ISSUE TITLE
What to do with these broken hackcoop links?

### ISSUE THREAD

**Issue by LukeShu** [LibreVPN/librevpn.org#1]
These are broken URL's in the LibreVPN docs/site that I'm not sure
what to do with.  Obviously, if there's an archive of the hackcoop
wiki (content dump -> github wiki?), I'd use that for the wiki links.
But what about the vpn@hackcoop.com.ar mailing list?  Or the IRC and
lab links?

    http://irc.hackcoop.com.ar/?channels=lab
    http://lab.hackcoop.com.ar/projects/librevpn/wiki/Tinc2dot
    http://listas.hackcoop.com.ar/listinfo/vpn
    vpn@hackcoop.com.ar

    http://wiki.hackcoop.com.ar/Categoría:LibreVPN
    http://wiki.hackcoop.com.ar/Categoría:LibreVPN/en
    http://wiki.hackcoop.com.ar/Correo_distribuido
    http://wiki.hackcoop.com.ar/Correo_local
    http://wiki.hackcoop.com.ar/Galponeo
    http://wiki.hackcoop.com.ar/LibreVPN:Crear
    http://wiki.hackcoop.com.ar/LibreVPN:IPv6

**Comment by fauno** [comment]
nope, those others are all dead.  i can recover the wiki articles from the backup i have

**Comment by fauno** [comment]
i was thinking on converting this site to jekyll and recover the wiki
articles as posts (i have a backup)

-- 
http://utopia.partidopirata.com.ar/
